src/router.ts

```diff
@@ -35,7 +56,8 @@ const spaceRoutes = [
   {
     path: 'create/:sourceProposal?',
     name: 'spaceCreate',
-    component: SpaceCreate
+    component: SpaceCreate,
+    children: spaceCreateSteps
```

Comment:
This might be a matter of taste/preference but I'd personally not use this variable and instead just put the array there directly. For me that feels cleaner.